static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,125 @@
-.email-composition-dialog {
-  .email-composition-contents {
-    display: flex;
-    flex-direction: column;
+.email-composition-dialog-wrapper {
```

Comment:
oh sorry, I got confused! It looks like this is copied to your other PR too, I wasn't sure which one I was looking at >_<